File: submissions/sm_110_mihir/week_3/day_6/assignments/pyramidPlay.js
function pyramid(level){
    var pos=0,rowCounter=0,starNumbers
    stars=''
    for(row=level;row>0;row--){
        rowCounter++
        // position of '0' in one row
        pos=row
        starNumbers=0
        for(col=0;col<level*2+1;col++){
            // print zero at position row Number times
            if(col==pos && starNumbers<rowCounter){
                stars+='0'
                starNumbers++
                if(rowCounter!=1)
                    pos=pos+2
            }
            else
                stars+='*'
        }
        console.log(stars)
        stars=''
    }
}


function inversePyramid(level){
    var pos=0,rowCounter=level+1,starNumbers
    stars=''
    for(row=1;row<=level;row++){
        rowCounter--
        // position of '0' in one row
        pos=row
        starNumbers=0
        for(col=0;col<level*2+1;col++){
            // print zero at position rowCounter times
            if(col==pos && starNumbers<rowCounter){
                stars+='0'
                starNumbers++
                if(rowCounter!=1)
                    pos=pos+2
            }
                
            else
                stars+='*'
        }
        console.log(stars)
        stars=''
    }
}

function rhombus(level){
    pyramid(level)
    inversePyramid(4)
}


rhombus(4)

File: submissions/sm_110_mihir/week_13/day_5/pyramid_play.py
def pyramid(level):
    pos=0
    row=0
    stars=""
    for i in range(level,0,-1):
        row+=1
        numStar=0
        pos=i
        for col in range(0,level*2+1):
            if col == pos and numStar<row:
                stars+='0'
                numStar+=1
                if row !=1:
                    pos=pos+2
            else:
                stars+= '*'
        print(stars)
        stars=""

def inverse(level):
    pos=0
    row=level+1
    stars=""
    for i in range(1,level+1):
        row-=1
        pos=i
        starNum=0
        for col in range(0,level*2+1):
            if col == pos and starNum<row:
                stars+="0"
                starNum+=1
                if row !=1:
                    pos += 2
            else:
                stars+="*"
        print(stars)
        stars=""

def rhombus(level):
    pyramid(level)
    inverse(level)

level= int(input("Enter level: "))
print("pyramid")
pyramid(level)
print("inverse pyramid")
inverse(level)
print("Rhombus")
rhombus(level)